src/main/java/com/google/sps/data/SampleData.java

```diff
@@ -172,4 +172,16 @@ public static void addToStudentScheduledSessionsByEmail(String email, TutorSessi
         students.add(student);
     }
 
+    public static void rateTutorByEmail(String tutorEmail, String studentEmail, int newRating) {
+        Tutor tutor = getTutorByEmail(tutorEmail);
+        Student student = getStudentByEmail(studentEmail);
+
+        tutors.remove(tutor);
+        tutor.rateTutor(newRating);
+        tutors.add(tutor);
+
+        students.remove(student);
+        student.markTutoringSessionAsRatedByTutorEmail(tutorEmail, newRating);
+        students.add(student);
```

Comment:
Same thing here. I don't think you have to remove and add the student again because you're just updating the student object.

Reply:
The reason why I was doing this was because of the limitations of the Arrays but now that they will be ArrayLists I think this should be much easier. I will update the functions after Arrays lists has been merged then.

Reply:
Just committed changes that update these functions